### PR TITLE
refactor: docstring for method returning RecordsCppImpl and TracePointData

### PR DESCRIPTION
**Pull request by h-suzuki-isp**
## Description
Fix docstring for method returning RecordsCppImpl and TracePointData

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] (Optional) The PR has been properly tested with CARET_report verification.
- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/infra/lttng/records_source.py [hunk 1]
```diff
@@ -704,7 +704,7 @@ def path_beginning_records(self) -> RecordsInterface:
             - callback_start_timestamp
             - rclcpp_publish_timestamp
             - callback_object
-            - publisher_object
+            - publisher_handle
```
[the remark this answers is not in the document]
Comment (h-suzuki-isp) [reply src/caret_analyze/infra/lttng/records_source.py:707]: Fixed in https://github.com/tier4/caret_analyze/pull/528/commits/f47d82123067fbb19ed17ce39aa671786cb614d8.

File: src/caret_analyze/infra/lttng/records_provider_lttng.py [hunk 2]
```diff
@@ -92,6 +92,15 @@ def communication_records(
...
             - [callback_name]/callback_start_timestamp
+        -------
+
+        Returns (intra proc communication)
+        -------
+        RecordsInterface
+            Columns
+
+            - [topic_name]/rclcpp_publish_timestamp
+            - [callback_name]/callback_start_timestamp
```
[the remark this answers is not in the document]
Comment (h-suzuki-isp) [reply src/caret_analyze/infra/lttng/records_provider_lttng.py]: Fixed in https://github.com/tier4/caret_analyze/pull/528/commits/66ae80de1eb7256a5f0945b9b08b8235ad0dc155.

File: src/caret_analyze/infra/lttng/lttng_info.py [hunk 3]
```diff
@@ -894,17 +895,18 @@ def subscription_callbacks(self) -> TracePointData:
         Returns
         -------
         pd.DataFrame
+
             columns
```
Comment (ymski) [inline src/caret_analyze/infra/lttng/lttng_info.py:899]:
```suggestion
            columns

```

File: src/caret_analyze/infra/lttng/lttng_info.py [hunk 4]
```diff
@@ -961,7 +966,9 @@ def publishers(self) -> TracePointData:
         Returns
         -------
         pd.DataFrame
+
             Columns
```
Comment (ymski) [inline src/caret_analyze/infra/lttng/lttng_info.py:970]:
```suggestion
            Columns

```

**Review by ymski — APPROVED** [review]
LGTM

**Pull request merged**